ksqldb-functional-tests/src/test/resources/query-validation-tests/fk-join.json

```diff
@@ -186,87 +184,502 @@
       "statements": [
         "CREATE TABLE left_table (id1 BIGINT PRIMARY KEY, f1 BIGINT) WITH (kafka_topic='left_topic', format='JSON');",
         "CREATE TABLE right_table (id2 BIGINT PRIMARY KEY, f2 BIGINT) WITH (kafka_topic='right_topic', format='JSON');",
-        "CREATE TABLE output AS SELECT id1, id2, f1, f2 FROM left_table FULL OUTER JOIN right_table ON id1 = f2;"
+        "CREATE TABLE output AS SELECT id1, f1, f2 FROM left_table AS lt JOIN right_table AS rt ON lt.id1 = rt.f2;"
       ],
       "expectedException": {
         "type": "io.confluent.ksql.util.KsqlStatementException",
-        "message": "Invalid join condition: table-table joins require to join on the primary key of the right input table. Got LEFT_TABLE.ID1 = RIGHT_TABLE.F2."
+        "message": "Invalid join condition: table-table joins require to join on the primary key of the right input table. Got LT.ID1 = RT.F2."
       }
     },
     {
-      "name": "Should fail on right non-key attribute for inner-join with qualifiers - feature flag enabled",
+      "name": "value column - inner join",
       "properties": {
         "ksql.joins.foreign.key.enable": true
       },
-      "comments": [
-        "TODO: this test should be deleted once the feature is ungated and the feature flag is removed",
-        "as it duplicates another test above (without the feature flag)"
+      "statements": [
+        "CREATE TABLE left_table (L_ID BIGINT PRIMARY KEY, NAME varchar, VALUE bigint) WITH (kafka_topic='left_topic', value_format='JSON');",
```

Comment:
nit: the formatting if rather confusing. Can we use all-caps for keywords and lower cases for everything else? Would be much easier to read...
`CREATE TABLE left_table (l_id BIGINT PRIMARY KEY, name VARCHAR, value BIGINT)`

Also applies to all other newly added tests.

Reply:
Went through and updating the casing of all statements to be consistent.